libs/pmp-web/shared/ui-sidebar/src/lib/components/sidebar/sidebar.component.html

```diff
@@ -5,20 +5,26 @@
         class="sidebar__item"
         [routerLink]="['/reviewer']"
         routerLinkActive="sidebar__item--active"
+        matTooltip="PR reviewers pending"
```

Comment:
Leave just Reviewers

Reply:
Please replace also Repository users with Reviewers in the title under the /reviewers URL.